### ISSUE TITLE
Dévérouillage ajout accessoire, après avoir un peu blindé les choses (help_text et RegExp)

### ISSUE THREAD

**Issue by GwilhermPoullennec** [inventaire-des-orgues/portail#369]

**Comment by patou** [comment]
Je sais pas si il faut permettre de pouvoir le faire ?
Si on veut garder la cohérence des données de référence.

**Comment by GwilhermPoullennec** [comment]
Oui, grâce à tous tes ajouts, je pense qu'on devrait pouvoir l'éviter et conserver ainsi une bonne qualité de données, car pour les accessoires les utilisateurs seront encore plus créatifs que pour les jeux
Il faudrait éviter les accessoires de type : "petite cuillère noire un peu tordue en bas à droite pour appeler la trompette, un peu fausse d'ailleurs"
:-)

**Comment by patou** [comment]
> Oui, grâce à tous tes ajouts, je pense qu'on devrait pouvoir l'éviter et conserver ainsi une bonne qualité de données, car pour les accessoires les utilisateurs seront encore plus créatifs que pour les jeux
> Il faudrait éviter les accessoires de type : "petite cuillère noire un peu tordue en bas à droite pour appeler la trompette, un peu fausse d'ailleurs"
> :-)

Oui, déjà dans la liste de l'inventaire, on a déjà plein de façon différente d'avoir accouplement pos/go

Par contre, on peut peut être avoir un formulaire de contact spécifique pour demander l'ajout de donnée (demande juste le nom à ajouter...)

**Comment by patou** [comment]
On peut aussi ajouter un champ de commentaire en dessous de Accessoire pour permettre d'ajouter des informations.
Par exemple le tutti introduit également l'octave aigue

**Comment by GwilhermPoullennec** [comment]
Texte : "Prenez vos précautions"
